### PR TITLE
examples: Update features/encryption/README.md file

### PR DESCRIPTION
**Pull request by AntiBargu**
When I read the above document, I found several small mistakes. In order to prevent misunderstandings among newcomers while reading. I fix those mistakes and add instructions for key files in the mTLS example.

RELEASE NOTES: none

### CODE REVIEW

File: examples/features/encryption/README.md [hunk 1]
```diff
@@ -91,6 +90,15 @@ successfully up.
...
 
+In this example, we use the public/private keys created ahead: 
+
+* "server_cert.pem" contains the server certificate (public key). 
+* "server_key.pem" contains the server private key. 
+* "ca_cert.pem" contains the certificate (certificate authority) that can verify the server's certificate.
+* "client_cert.pem" contains the client certificate (public key). 
+* "client_key.pem" contains the client private key. 
+* "client_ca_cert.pem" contains the certificate (certificate authority) that can verify the client's certificate.
+
```
Comment (zasweq) [inline examples/features/encryption/README.md:101]: Nit: please wrap to 80 chars.

File: examples/features/encryption/README.md [hunk 2]
```diff
@@ -91,6 +90,15 @@ successfully up.
 In mutual TLS (mTLS), the client and the server authenticate each other. gRPC
 allows users to configure mutual TLS at the connection level.
 
+In this example, we use the public/private keys created ahead: 
+
+* "server_cert.pem" contains the server certificate (public key). 
+* "server_key.pem" contains the server private key. 
+* "ca_cert.pem" contains the certificate (certificate authority) that can verify the server's certificate.
```
Comment (zasweq) [inline examples/features/encryption/README.md:97]: "the certificate (certificate authority)"? Perhaps the certificate of the certificate authority?

File: examples/features/encryption/README.md [hunk 3]
```diff
@@ -91,6 +90,15 @@ successfully up.
 In mutual TLS (mTLS), the client and the server authenticate each other. gRPC
 allows users to configure mutual TLS at the connection level.
 
+In this example, we use the public/private keys created ahead:
```
Comment (zasweq) [inline examples/features/encryption/README.md:93]: Nit: ahead of time.

**Review by zasweq — APPROVED** [review]
LGTM outside minor nits.

**Comment by zasweq** [comment]
Passing to Doug for a second pass.

File: examples/features/encryption/README.md [hunk 4]
```diff
@@ -1,7 +1,6 @@
 # Encryption
 
-The example for encryption includes two individual examples for TLS and ALTS
-encryption mechanism respectively.
+The example for encryption includes three individual examples for TLS, ALTS and mTLS encryption mechanism respectively.
```
Comment (dfawley) [inline examples/features/encryption/README.md:3]: Please wrap to 80 columns (whole file).

File: examples/features/encryption/README.md (hunk 2 above)
Comment (dfawley) [reply examples/features/encryption/README.md:97]: "the certificate _from_ the certificate authority"?

Comment (AntiBargu) [reply examples/features/encryption/README.md:97]: Yes

File: examples/features/encryption/README.md (hunk 4 above)
Comment (AntiBargu) [reply examples/features/encryption/README.md:3]: OK

File: examples/features/encryption/README.md (hunk 2 above)
Comment (AntiBargu) [reply examples/features/encryption/README.md]: > "the certificate (certificate authority)"? Perhaps the certificate of the certificate authority?

yes

**Review by AntiBargu — COMMENTED** [review]
Review completed

**Review by dfawley — APPROVED** [review]

**Comment by dfawley** [comment]
Assigning to @zasweq to re-review and merge if OK.

**Review by zasweq — APPROVED** [review]

**Pull request merged**